### PR TITLE
docs: initial readme draft

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)

Closes #XXXXX

This will display on our organisation page at https://github.com/freeCodeCamp

Not sure what else we want to add here - cc'ing @QuincyLarson for the copy.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# freeCodeCamp
+
+<img src="https://cdn.freecodecamp.org/platform/universal/fcc_meta_1920X1080-indigo.png" align="center" alt="freeCodeCamp's logo" />
```
Comment (raisedadead) [inline profile/README.md:3]: This is a pretty gargantuan image, I would recommend something smaller in height. I am of the opinion that the twitter header we have (same image on the main repo) is the most appealing - even though the logo is old.

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+You can [read more about freeCodeCamp](https://freecodecamp.org/news/about).
+
+## Our Curriculum
+
+The codebase for our primary `/learn` platform can be found in the [freeCodeCamp repository](https://github.com/freeCodeCamp/freeCodeCamp), which includes our curriculum lessons, web client, and API server.
+
+## Contributing
+
+If you are interested in contributing to freeCodeCamp, we recommend starting with our [contributing documentation](https://contribute.freecodecamp.org).
```
Comment (raisedadead) [inline profile/README.md:17]: I think we should remove these. Instead, let's use this feature to have a call for sponsors. I would use the copy and the list of projects from out GitHub sponsors page

Comment (raisedadead) [reply profile/README.md:17]: If you go that route this is a cool tool to generate a SVG that we can reference in the content: https://github.com/antfu/sponsorkit

File: profile/README.md (hunk 1 above)
Comment (nhcarrigan) [reply profile/README.md]: Replaced!

File: profile/README.md (hunk 2 above)
Comment (nhcarrigan) [reply profile/README.md]: I went ahead and added the sponsors copy.

**Review by QuincyLarson — COMMENTED** [review]
This looks great. A couple quick numeric changes:

8,000+ coding tutorials
1,000+ free YouTube courses...

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+You can [read more about freeCodeCamp](https://freecodecamp.org/news/about).
+
+## Become a Sponsor to freeCodeCamp
+
+In 2014, a teacher started an open source community called freeCodeCamp.org. This community has grown to include millions of developers from around the world. Together, we have built:
+
+💻 A free 3,000-hour programming curriculum
+🏛️ Forums and chat rooms where you can get coding help and feedback on your projects
+🧪 6,000+ coding tutorials
+🎒 100s of free YouTube courses on Computer Science, Data Science, and Machine Learning
```
Comment (nhcarrigan) [inline profile/README.md:18]:
```suggestion
🧪 8,000+ coding tutorials
🎒 1,000+ free YouTube courses on Computer Science, Data Science, and Machine Learning

```

**Review by raisedadead — APPROVED** [review]

**Pull request merged**